.circleci/config.yml

```diff
@@ -191,6 +191,42 @@ jobs:
           command: |
             pnpm --dir libs/ballot-encoder test
 
+  test-libs-cdf-schema-builder:
+    executor: nodejs
+    resource_class: xlarge
+    steps:
+      - checkout-and-install
+      - run:
+          name: Build
+          command: |
+            pnpm --dir libs/cdf-schema-builder build
+      - run:
+          name: Lint
+          command: |
+            pnpm --dir libs/cdf-schema-builder lint
+      - run:
+          name: Test
+          command: |
+            pnpm --dir libs/cdf-schema-builder test
+
+  test-libs-cdf-types-election-event-logging:
```

Comment:
curious why you are making this specific to the file type as opposed to a more general cdf types library? 

It would be more annoying for us but I could see an argument for them to be cdf-specific, and have it live as a separate repo outside of vxsuite as it could be theoretically useful to someone else. But I'm not sure its worth the overhead of that.

Reply:
After I wrote this I realized there could be type name conflicts between the generated files and its probably easier to avoid that this way.

Reply:
That's part of it. NIST breaks it into multiple parts (i.e. logging, reporting, etc) so I followed suit.